errors/errors.go

```diff
@@ -37,3 +40,15 @@ func WriteHTTPUnsupportedMediaType(w http.ResponseWriter, msg string, err error)
 func WriteHTTPInternalServerError(w http.ResponseWriter, msg string, err error) apiError {
 	return writeHttpError(w, msg, http.StatusInternalServerError, err)
 }
+
+func WriteHTTPBadBodySchema(where string, w http.ResponseWriter, errors []gojsonschema.ResultError) apiError {
+	sb := strings.Builder{}
+	sb.WriteString("Body validation error in ")
```

Comment:
I don't see people use the String Builder very often in Go, normally stuff like this is done with a `fmt.Sprintf`